README.md

```diff
@@ -1,3 +1,16 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
```

Comment:
I would put this after the badges and main description of this repository (before overview header), also the readme is not so long, so not much value in TOC (imo)

Reply:
591abed05b49799ae0dcb6300ea7d497eb0c1323